README.md

```diff
@@ -126,7 +138,58 @@ const sendBundleResponse = await skipBundleClient.sendBundle(signedBundle, DESIR
 ```
 
 `DESIRED_HEIGHT_FOR_BUNDLE` should be a number, where 0 asks the Sentinel to autodetermine the next height.
+### Terra and others using terra.js / feather.js
+Terra and [alliance](https://github.com/terra-money/alliance) chains usually use an alternative client library called [terra.js](https://github.com/terra-money/terra.js), it was rebranded to [feather.js](https://github.com/terra-money/feather.js) in late 2022 to add multichain support (i.e. similar to cosmjs, you can use it to interact with not only terra but also other cosmos SDK chains, most likely alliance chains that share security with Terra).
```

Comment:
nits:
Can you capitalize `Alliance`?
And remove the history and other information about `feather.js`? It isn't really relevant to this repo.

I'd like to replace this line with just:
 > Terra and [Alliance](https://github.com/terra-money/alliance) chains use an alternative client library called [feather.js](https://github.com/terra-money/feather.js).